api/database/factories/AssessmentStepFactory.php

```diff
@@ -25,9 +25,21 @@ public function definition()
     {
         $title = $this->faker->name();
 
+        // some assessment types are special and have controlled handling, do not generate in factory/seeding
+        $validAssessmentStepTypes = [
```

Comment:
Prevent seeding from setting up pools with multiple assessments of type application screening or screening question.